lms/djangoapps/survey/models.py

```diff
@@ -15,6 +15,7 @@
 from survey.exceptions import SurveyFormNameAlreadyExists, SurveyFormNotFound
 
 log = logging.getLogger("edx.survey")
+app_label = 'survey'
```

Comment:
Most all the other app_labels in the platform are defined within the model itself, using the Meta class:
```
    class Meta(object):
        app_label = 'survey'
```
Is there a reason why it's placed in the top-level module here?

Reply:
Simply because this is the way I'd done this conversion before. It works but after checking the docs it's clearly not how it's intended to be done, I'll update it.